packages/gatsby-cli/src/structured-errors/error-map.js

```diff
@@ -81,6 +81,15 @@ const errorMap = {
     type: `CONFIG`,
     level: `ERROR`,
   },
+  "10126": {
+    text: context =>
+      `A ${
+        context.configName
+      } that exports a function can only be used as a theme and not run directly.\nIf you are trying to run a theme directly, use the theme in an example site or starter instead and run that site to test.\nIf you are in the root gatsby-config.js for your site, change the export to be an object and not a function as functions are not supported in the root gatsby-config.`,
```

Comment:
This is a huge giant line with newline characters because if I just move them to the next line manually I get indentation. I could not use `report.stripIndent` while using structured errors as far as I could tell